community/virtualbox-guest-modules-vanilla/APKBUILD

```
@@ -1,9 +1,9 @@
 # Maintainer: Ben Allen <bensallen@me.com>
 # Contributor: Ben Allen <bensallen@me.com>
 
-# when chaning _ver we *must* bump _rel
+# when changing _ver we *must* bump _rel
 _name=virtualbox-guest-modules
-_ver=6.0.8
+_ver=6.0.10
 _rel=0
```

As comment says, we need to increase _rel.

````suggestion
_rel=1
````